CONTRIBUTING.md

```diff
@@ -244,7 +244,21 @@ The following example is for releasing a patch for `v0.69.0` -> `v0.69.1`.
 - Create PRs with base set to that release branch
   - When the PR is merged, a changeset PR is created
   - When the changeset PR is merged into the release branch, the next patch version is released and the commit is tagged (e.g. `v0.69.1`)
-- After release, delete the release branch from GitHub
+- After release, the release branch will be automatically deleted
```

Comment:
When patching the _latest release_, this last step could be to open a new PR using the same branch but against `master`. This would close the gap and re-pass the updates to the ongoing changesets PR (and upcoming release).

Reply:
Do you mean that we automate it in the CI or just explain it to the user to do it manually?
In both cases we'd have to have an exception to not delete the release branch when it's a patch to the latest release.

I touched upon automation in [this comment](https://github.com/FuelLabs/fuels-ts/pull/2497/files#r1636888919), but after a bit of investigation it doesn't seem like a particularly hard thing to automate. Initially I thought that it's not possible to open a PR when it's got merge conflicts with the base branch, but I just took a look on our repo with some old branches (e.g. trying to create a new PR from https://github.com/FuelLabs/fuels-ts/tree/aa/feat/new-abi-spec to `master`, which has merge conflicts) and it seems possible. Do you see any potential blockers to automate this?

Reply:
No, I don't. But if that's too much trouble, we could simply add some instructions.

 1. Clone your released tag branch with `git <...>`
    > If the branch was deleted, they'll have to recreate it.
 3. Create PR against `master`, etc.

Reply:
@arboleya I have automated it [here](https://github.com/FuelLabs/fuels-ts/pull/2497/files#diff-e426ed45842837026e10e66af23d9c7077e89eacbe6958ce7cb991130ad05adaR174) and updated the `CONTRIBUTING.md` file accordingly.